common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -45,18 +48,36 @@
                 <div class="@RenderClasses(Map("youtube-media-atom__overlay" -> true, "vjs-big-play-button" -> !expired))" style="background-image: url(@Video700.bestFor(image))">
 
                 @if(!expired) {
-                    @defining(!cardStyle.exists(c => c.toneString != "media" && !playable)) { showPlayButton: Boolean =>
-                        @if(showPlayButton) {
-                            <div class="youtube-media-atom__play-button vjs-control-text">Play Video</div>
+                    @if(immersiveMainMedia) {
+                        @defining(!cardStyle.exists(c => c.toneString != "media" && !playable)) { showPlayButton: Boolean =>
```

Comment:
You don't need this check for `showPlayButton` in this case because in the immersiveMedia case you always want to show the play button.